lang/lang-es.js

```diff
@@ -0,0 +1,560 @@
+    Cpoll: 'Encuesta del curso',
+    Cquiz: 'Cerrar cuestionario',
+    createnewpoll: 'Crear nueva encuesta',
+    Creator: 'Creador',
+    comment: 'Comentarios',
+    dap: 'Falso',
+    delete: 'Eliminar',
+    delblank: 'Eliminar la opción en blanco',
+    Delete: 'Eliminar',
+    DevicesNotFoundError: 'La webcam no se encuentra disponible, o no está conectada correctamente.',
+    deletepopup: '¿Estás seguro de eliminar? ?',
+    disable: 'Desactivar',
+    disabled: 'Desactivar',
+    disableAllVideo: 'Desactivar todo el video',
+    dltDisabled: 'Puede ser eliminado por el creador de la encuesta.',
+    documentShare: 'Compartir documento',
+    DocumentSharing: 'Compartiendo documentos',
+    DocumentSharedbHeading: 'Panel de documentos',
+    docUploadSuccess: 'Documento cargado exitosamente',
+    download: 'Descargar',
+    downloadFile: 'Descargar archivo',
+    downloadsession: 'Por favor espere mientras se descarga la grabación',
+    drawArea: 'Área de dibujo',
+    dropfilehere: 'Suelta los archivos aquí',
+    duplicateUploadMsg: 'Este archivo ya existe, cargue un archivo nuevo.',
+    edit: 'Editar',
+    editorCode: 'Editor de código',
+    edittitle: 'Editar título',
+    editorRichDisable: 'Modo de escritura',
+    editorRichEnable: 'Solo lectura',
+    educator: 'Educador',
+    EHTMLPresentUrl: 'Ingrese la URL de presentación HTML5',
+    enable: 'Activar',
+    enablehistory: '"Historial del navegador" debe habilitarse durante la sesión.',
+    enableAllVideo: 'Activar todo el video',
+    enteryouryoutubeurl: 'Ingrese la URL de Youtube',
+    etDisabledA: 'No se puede editar, intento de encuesta',
+    etDisabledCr: 'Puede ser editado por el creador de la encuesta.',
+    ETime: 'Tiempo transcurrido',
+    TimeRemaining: 'Tiempo restante',
+    QuestionsOverview: 'Resumen de preguntas',
+    UsersAttempted: 'Los usuarios intentaron',
+    gRname: 'Nombre',
+    gRtimeTaken: 'Tiempo tomado',
+    gRgrade: 'Grado',
+    gRQAttempted: 'Q. intento',
+    gRCorrect: 'Correcto' ,
+    FF2: 'Avance rápido 2',
+    FF8: 'Avance rápido 8',
+    filenotsave: 'Su archivo no se pudo guardar.',
+    filetsaveTS: 'La sesión está lista para guardarse.',
+    Finish: 'Finalizado',
+    fitToPage: 'Ajustar a la página',
+    fitToScreen: 'Ajustar la pantalla',
+    Freehand: 'A mano',
+    general: 'General',
+    Greport: 'Informe de calificaciones',
+    highBandWidthSpeed: 'Su ancho de banda es lo suficientemente bueno.',
+    httpsmissing: 'Solo se permiten orígenes seguros (https) para compartir la pantalla.',
+    ieBrowserIssue: 'Internet Explorer no es compatible, Congrea es totalmente compatible con Chrome y Firefox.',
+    ios7support: 'Para Apple, Virtual Class es compatible con iOS 8 y versiones superiores.',
+    iosAudEnable: 'Toque aquí para habilitar el audio.',
+    InternalError: 'Asegúrese de que no se esté utilizando la misma cámara web <br /> en varios navegadores o varias aplicaciones. ',
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This line has a length of 133. Maximum allowed is 120.](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=5529949)